pkg/sensors/exec/userinfo/userinfo.go

```diff
@@ -37,6 +38,7 @@ func MsgToExecveAccountUnix(m *exec.MsgExecveEventUnix) error {
 	if option.Config.UsernameMetadata == int(option.USERNAME_METADATA_UNIX) {
 		username, err := getAccountUnix(m.Unix.Process.UID, &m.Unix.Msg.Namespaces)
 		if err != nil {
+			errormetrics.ErrorTotalInc(errormetrics.ProcessMetadataUsernameFailed)
```

Comment:
Shouldn't we check here if the error is ErrNotInHostNs?

Reply:
Makes sense, I made it 'process_metadata_username_ignored_not_in_host_namespaces' a bit too long but it has to be descriptive anyway.